src/utils/GcTcpHub.hxx

```diff
@@ -60,16 +60,28 @@ public:
         return tcpListener_.is_started();
     }
 
+    /// @return currently connected client count.
+    unsigned get_num_clients()
+    {
+        return numClients_;
```

Comment:
do we need an AtomicHolder here?

Reply:
I don't think it's super meaningful. It would be necessary if the data type of numClients_ were thread-compatible but not thread-safe for the read accessor that we are using.

Reply:
Perhaps it isn't necessary but there is an AtomicHolder usage for increment/decrement of this same var.

Reply:
That's needed because it is a write access. A primitive type is thread-compatible but not thread-safe for writes.

Reply:
Ahh, yes that is correct. We can ignore this here then.